README.md

```diff
@@ -210,3 +219,6 @@ un-listed error, simply execute `run.py` again with the same exact options.
 #### Cloudformation
 On occasion the Cloudformation stack will fail to delete properly. Simply go
 into the AWS console and find the Cloudformation stack and delete it.
+
+#### When all hope is lost
+Most likely it's because you have existing settings in your AWS region.  Pick another AWS region and start over. 
```

Comment:
I thought this was the troubleshooting section.   Also, I couldn't get it to work in N. California AWS.   After trying to get it to work, I gave up.   I just picked another region and it started to work.   People need to know this or else you and your team is going to do support for all of us.